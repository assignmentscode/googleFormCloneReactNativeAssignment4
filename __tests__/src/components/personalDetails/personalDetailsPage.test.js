import React from 'react';
import renderer from 'react-test-renderer';
import PersonalDetailsPage from '../../../../src/components/personalDetails/personalDetailsPage';

describe('PersonalDetailsPage', () => {
  it('snapshot should match the previous snapshot', () => {
    const tree = renderer.create(<PersonalDetailsPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
