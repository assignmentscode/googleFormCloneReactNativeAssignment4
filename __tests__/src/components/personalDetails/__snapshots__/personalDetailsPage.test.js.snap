// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalDetailsPage snapshot should match the previous snapshot 1`] = `
<View>
  <Text>
     PersonalDetailsPage 
  </Text>
</View>
`;
